.github/workflows/ci.yml

```diff
@@ -261,6 +261,7 @@ jobs:
         target:
           - powerpc-unknown-linux-gnu
           - powerpc64-unknown-linux-gnu
+          - powerpc64-ibm-aix
```

Comment:
Unfortunately, this fails with the following error:
```
$ rustup toolchain install stable --target powerpc64-ibm-aix --profile minimal --no-self-update
info: syncing channel updates for 'stable-x86_64-unknown-linux-gnu'
info: latest update on 2023-02-09, rust version 1.67.1 (d5a82bbd2 2023-02-07)
error: component 'rust-std' for target 'powerpc64-ibm-aix' is unavailable for download for channel 'stable'
If you don't need the component, you could try a minimal installation with:

    rustup toolchain add stable --profile minimal
```
Perhaps adding this to CI is not possible today?

Reply:
Ah, I forgot about it. I'll add it back once `powerpc64-ibm-aix` is available in rustup.